src/ensembl/src/content/app/species/SpeciesPage.scss

```diff
@@ -1,5 +1,9 @@
 @import 'src/styles/common';
 
+.speciesPage {
+  height: 100%;
```

Comment:
This have introduced an extra scrollbar to the right which is always visible

Reply:
The scrollbar turns out to be the result of the not overridden margin bottom of the species component.  

![image](https://user-images.githubusercontent.com/6834224/125201893-fe8c0f00-e268-11eb-9892-4197e29253e5.png)

Reply:
Made some updates to the layout that should account for that.